Obsidian.API/Plugins/Services/IFileReader.cs

```diff
@@ -76,5 +76,13 @@ public string[] GetSubdirectories(string path)
 
             return Directory.GetDirectories(path);
         }
+
+        /// <summary>
+        /// Creates a directory, that is used by default for relative paths.
+        /// </summary>
+        /// <param name="createOwnDirectory">If set to <b><c>false</c></b>, the automatically assigned directory for your plugin will be skipped.</param>
+        /// <param name="skipFolderAutoGeneration">If set to <b><c>true</c></b>, skips the auto generation method for default plugin dir. Also, <b><c>createOwnDirectory</c></b> needs to be <b><c>true</c></b> for this to work.</param>
+        /// <returns>Path to the created directory.</returns>
+        public string CreateWorkingDirectory(bool createOwnDirectory = true, bool skipFolderAutoGeneration = false);
```

Comment:
Could this definition (and the one in IFileWriter) be moved to IFileService?

Reply:
i think it can be moved. I'll check that.

Reply:
moved